packages/client/src/v2-events/components/DownloadButton.tsx

```diff
@@ -212,6 +213,7 @@ export function DownloadButton({ id, className, event }: DownloadButtonProps) {
                   }
                 : undefined
             }
+            name={`${user?.name[0].given[0]} ${user?.name[0].family}`}
```

Comment:
Do we have a centralized way to format names? This was an issue in V1 that we did name formatting in multiple different places and it started to get difficult to track

Reply:
https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/v2-events/features/events/registered-fields/Name.tsx#L165